libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -71,19 +85,26 @@ export interface LuisRecognizerSettings {
  * This component can be used within your bots logic by calling [recognize()](#recognize).
  */
 export class LuisRecognizer {
-    private settings: LuisRecognizerSettings;
+    private application: LuisApplication;
+    private options: LuisPredictionOptions;
+    private includeApiResults: boolean;
+
     private luisClient: LuisClient;
     private cacheKey = Symbol('results');
 
     /**
      * Creates a new LuisRecognizer instance.
-     * @param settings Settings used to configure the instance.
+     * @param application LUIS application to use.
+     * @param options Options used to control predictions.
      */
-    constructor(settings: LuisRecognizerSettings) {
-        this.settings = Object.assign({}, settings);
+    constructor(application: LuisApplication, options?: LuisPredictionOptions, includeApiResults?: boolean) {
+        this.application = application;
+        this.options = Object.assign({}, options);
```

Comment:
>{} [](start = 37, length = 2)

If it simplifies things you can pre-set your option default values here. That way they'll only get changed if the caller assigns them.